### ISSUE TITLE
Providing policy_import_json with other arguments causes crash

### ISSUE THREAD

**Issue by olegmukhin** [F5Networks/terraform-provider-bigip#766]
### Environment

 * TMOS/Bigip Version: 16.1.3
 * Terraform Version: 1.3.7
 * Terraform bigip provider Version: 1.16.2

### Summary

Setting the open_api_files and/or server_technologies arguments AND providing an existing WAF policy via policy_import_json causes provider to crash.

### Steps To Reproduce

1. Set open_api_files and/or server_technologies arguments of the bigip_waf_policy resource.
1. Provide a valid WAF policy via policy_import_json to the same bigip_waf_policy resource.

### Expected Behavior
WAF policy deployment.

### Actual Behavior
Outcome is the following error:

```
bigip_waf_policy.waf-policy: Creating...

╷

│ Error: Plugin did not respond

│

│   with bigip_waf_policy.waf-policy,

│   on [main.tf](http://main.tf/) line 24, in resource "bigip_waf_policy" "waf-policy":

│   24: resource "bigip_waf_policy" "waf-policy" {

│

│ The plugin encountered an error, and failed to respond to the

│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may

│ contain more details.

╵

Stack trace from the terraform-provider-bigip_v1.16.2 plugin:

panic: runtime error: index out of range [0] with length 0

goroutine 113 [running]:

[github.com/F5Networks/terraform-provider-bigip/bigip.getpolicyConfig(0xc0003dd528](http://github.com/F5Networks/terraform-provider-bigip/bigip.getpolicyConfig(0xc0003dd528)?)

                [github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_awaf_policy.go:650](http://github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_awaf_policy.go:650) +0x2ad4

[github.com/F5Networks/terraform-provider-bigip/bigip.resourceBigipAwafPolicyCreate(0xe1d0a0](http://github.com/F5Networks/terraform-provider-bigip/bigip.resourceBigipAwafPolicyCreate(0xe1d0a0)?, {0xf30c80?, 0xc000a537a0})

                [github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_awaf_policy.go:274](http://github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_awaf_policy.go:274) +0x1da

[github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc00024b800](http://github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc00024b800), 0xc0005c2af0, 0xc0004b06e0, {0xf30c80, 0xc000a537a0})

                [github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/resource.go:305](http://github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/resource.go:305) +0x438

[github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc00024b900](http://github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc00024b900), 0xc0003dd818, 0xf3f1eb?, 0xf?)

                [github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/provider.go:294](http://github.com/hashicorp/terraform-plugin-sdk@v1.1.0/helper/schema/provider.go:294) +0x70

[github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc0005b21f8](http://github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc0005b21f8), {0xc0006027e0?, 0x59d806?}, 0xc0006027e0)

                [github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/helper/plugin/grpc_provider.go:885](http://github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/helper/plugin/grpc_provider.go:885) +0x805

[github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xef7320](http://github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(%7B0xef7320)?, 0xc0005b21f8}, {0x10fbc50, 0xc0004b2690}, 0xc000602780, 0x0)

                [github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/tfplugin5/tfplugin5.pb.go:3189](http://github.com/hashicorp/terraform-plugin-sdk@v1.1.0/internal/tfplugin5/tfplugin5.pb.go:3189) +0x170

[google.golang.org/grpc.(*Server).processUnaryRPC(0xc00005c480](http://google.golang.org/grpc.(*Server).processUnaryRPC(0xc00005c480), {0x10ffcc0, 0xc00005c600}, 0xc0003eac00, 0xc0001214a0, 0x16c53e0, 0x0)

                [google.golang.org/grpc@v1.23.0/server.go:995](http://google.golang.org/grpc@v1.23.0/server.go:995) +0xe1e

[google.golang.org/grpc.(*Server).handleStream(0xc00005c480](http://google.golang.org/grpc.(*Server).handleStream(0xc00005c480), {0x10ffcc0, 0xc00005c600}, 0xc0003eac00, 0x0)

                [google.golang.org/grpc@v1.23.0/server.go:1275](http://google.golang.org/grpc@v1.23.0/server.go:1275) +0xa16

[google.golang.org/grpc.(*Server).serveStreams.func1.1()](http://google.golang.org/grpc.(*Server).serveStreams.func1.1())

                [google.golang.org/grpc@v1.23.0/server.go:710](http://google.golang.org/grpc@v1.23.0/server.go:710) +0x98

created by [google.golang.org/grpc.(*Server).serveStreams.func1](http://google.golang.org/grpc.(*Server).serveStreams.func1)

                [google.golang.org/grpc@v1.23.0/server.go:708](http://google.golang.org/grpc@v1.23.0/server.go:708) +0xea

Error: The terraform-provider-bigip_v1.16.2 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely

helpful if you could report the crash with the plugin's maintainers so that it

can be fixed. The output above should help diagnose the issue.
```

**Comment by RavinderReddyF5** [comment]
Hi @olegmukhin fixed in v1.17.0

**Comment by olegmukhin** [comment]
@RavinderReddyF5 thank you - we tested and this works for us now.
